tests/Model/BasePHPElement.php

```diff
@@ -113,25 +115,34 @@ protected static function findTypeFromAttribute(array $attrGroups): ?string
         return null;
     }
 
-    protected static function findSinceVersionFromAttribute(array $attrGroups): ?string
+    protected static function findAvailableVersionsRangeFromAttribute(array $attrGroups): array
     {
+        $versionRange = [];
         foreach ($attrGroups as $attrGroup) {
             foreach ($attrGroup->attrs as $attr) {
                 if ($attr->name->toString() === PhpStormStubsElementAvailable::class) {
-                    $arg = $attr->args[0]->value;
-                    if ($arg instanceof Array_) {
-                        $value = $arg->items[0]->value;
-                        if ($value instanceof String_) {
-                            return $value->value;
+                    if (count($attr->args) == 2) {
+                        foreach ($attr->args as $arg) {
+                            $versionRange[$arg->name->name] = $arg->value->value;
                         }
                     } else {
-                        return $arg->value;
+                        $arg = $attr->args[0]->value;
+                        if ($arg instanceof Array_) {
+                            $value = $arg->items[0]->value;
+                            if ($value instanceof String_) {
+                                return ['from' => $value->value];
+                            }
+                        } else {
+                            $rangeName = $attr->args[0]->name;
+                            return $rangeName === null || $rangeName->name == 'from' ?
+                                ['from' => $arg->value, 'to' => '8.0'] :
```

Comment:
What if function will be declared with attribute `#[PhpStormStubsElementAvailable(from: '7.1', to: '8.0')]`. How should we compare '8.0' and 'latest'?

Reply:
Let's introduce a class `PHPVersions` which will contain a list of all existing versions from `Utils.php` and let's take `$versions[-1]` instead of '8.0`. That will reduce the number of places to update to the single source of truth.

Reply:
Yes, this is a good solution! Thank you for the suggestion. I'll fix this.